packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -348,6 +348,33 @@ export default class VTable {
                 }
                 break;
 
+            case TableOperation.MergeCells:

```

Comment:
Question: is it possible to just reuse MergeLeft and MergeAbove to do MergeCells? e.g.
case MergeCells:
  for (...) for (...) this.edit(MergeLeft) ;

Reply:
I put the common code between the merges function inside a function